examples/pytorch/gcn/README.md

```diff
@@ -2,33 +2,48 @@ Graph Convolutional Networks (GCN)
 ============
 
 - Paper link: [https://arxiv.org/abs/1609.02907](https://arxiv.org/abs/1609.02907)
-- Author's code repo: [https://github.com/tkipf/gcn](https://github.com/tkipf/gcn). Note that the original code is 
-implemented with Tensorflow for the paper. 
+- Author's code repo: [https://github.com/tkipf/gcn](https://github.com/tkipf/gcn). Note that the original implementation is based on Tensorflow.
 
-Dependencies
+Requirements
 ------------
-- PyTorch 0.4.1+
-- requests
-
-``bash
+```
 pip install torch requests
-``
+```
```

Comment:
`requests` have been included in DGL's dependency list  https://github.com/dmlc/dgl/blob/master/python/setup.py , so we could remove this section.

Reply:
Okay

Reply:
Okay.